src/Init/SimpLemmas.lean

```diff
@@ -114,6 +117,94 @@ end SimprocHelperLemmas
 @[simp] theorem not_false_eq_true : (¬ False) = True := eq_true False.elim
 @[simp] theorem not_true_eq_false : (¬ True) = False := by decide
 
+-- From Std.
```

Comment:
```suggestion
```
Do we need to track bits of history in comments like this? I suspect it will be more confusing than anything a year from now.